blocks/library/more/index.js

```diff
@@ -68,33 +68,54 @@ export const settings = {
 		],
 	},
 
-	edit( { attributes, setAttributes, isSelected } ) {
-		const { customText, noTeaser } = attributes;
-
-		const toggleNoTeaser = () => setAttributes( { noTeaser: ! noTeaser } );
-		const defaultText = __( 'Read more' );
-		const value = customText !== undefined ? customText : defaultText;
-		const inputLength = value.length ? value.length + 1 : 1;
-
-		return [
-			isSelected && (
-				<InspectorControls key="inspector">
-					<ToggleControl
-						label={ __( 'Hide the teaser before the "More" tag' ) }
-						checked={ !! noTeaser }
-						onChange={ toggleNoTeaser }
+	edit: class extends Component {
+		constructor() {
+			super( ...arguments );
+			this.onChangeInput = this.onChangeInput.bind( this );
+
+			this.state = {
+				defaultText: __( 'Read more' ),
+			};
+		}
+
+		onChangeInput( event ) {
+			this.setState( {
```

Comment:
Note: `this.setState` will _always_ trigger a render, even if the next state is effectively equal, unless the component is a [pure component](https://reactjs.org/docs/react-api.html#reactpurecomponent), and even then, it's better to avoid the unnecessary effort of a [shallow equality check](https://reactjs.org/docs/shallow-compare.html) by just not calling `this.setState` if not strictly necesssary.

Here, it's not so much an issue because if this code is reached, we assume a render will likely occur anyways because of the subsequent `this.props.setAttributes`. But something to be cautious of all the same.